tests/unit/readers/cubeconfig/v1_0/test_cube_from_config_json.py

```diff
@@ -208,7 +207,7 @@ def test_build_config_ok():
 
     col_observation = cube.columns[4]
     assert isinstance(
-        col_observation.structural_definition, QbMultiMeasureObservationValue
+        col_observation.structural_definition, QbObservationValue
```

Comment:
I'd also assert that the `measure` is None (as well as in other locations which used to assert that it was a `QbMultiMeasureObservationValue`)

Reply:
Again, I don't see the changes in this file.